.github/workflows/unit-tests.yml

```diff
@@ -80,7 +82,7 @@ jobs:
   # https://github.com/orgs/community/discussions/33579
   success:
     name: Tests successful
-    if: always()
+    if: (github.repository == 'edx/edx-platform-private') || (github.repository == 'openedx/edx-platform' && (startsWith(github.base_ref, 'open-release') == false))
```

Comment:
@kdmccormick This check fails when GH workers are used for tests (see [example failure](https://github.com/openedx/edx-platform/actions/runs/4944803620/jobs/8840800058)) so needs to be disabled in that case. I copied it from [this PR](https://github.com/openedx/edx-platform/pull/31310) that was merged into the Olive branch.

Reply:
Got it, thanks.

It's confusing because the comment above it is inaccurate: it's _not_ a required check for branch protection. If it were, then all PRs from forks or towards release branches would fail, since this required check would be skipped.

As far as I can tell, this check was *once* required, but now it is not, and now that it is not, it is a no-op.

I don't want to hold up Palm work with this, so this is good for now. I'll file a followup issue.